CHANGES.rst

```diff
@@ -1,14 +1,127 @@
+methods were implemented, which simulate binned counts maps and spectra from models.
+In addition a nice string representations for all of the dataset classes was implemented
+together with convenience functions to compute residuals using different methods on all
+of them. The algorithm and API of the current `LightCurveEstimator` was changed to
+use datasets. Now it is possible to compute lightcurves using spectral as well
+as cube based analyses. The definition of the position angle of the `SkyEllipse` model
+was changed to follow IAU conventions.
+
+The handling of sky regions in Gammapy was unified as described in `PIG 10`_.
+For convenience regions can now also be created from DS9 region strings. The clean-up
+process of `gammapy.spectrum` was continued by removing the `PHACountsSpectrum`
+class, which is now fully replaced by the `SpectrumDatasetOnOff` class. The
+`Energy` and `EnergyBounds` classes were also removed. Grids of energies can be
+created and handled directly using the `MapAxis` object now.
+
+The algorithm to compute solid angles for maps was fixed, so that it gives correct
+results for WCS projections even with high spatial distortions. Standard analyses
+using TAN or CAR projections are only affected on a <1% level. Different units
+for the energy axis of the counts and exposure map in a `MapDataset` are now
+handled correctly.
+
+The recommended conda environment for Gammapy v0.13 was updated. It now relies
+on Python 3.7, Ipython 7.5, Scipy 1.3, Matplotlib 3.1, Astropy 3.1, and Healpy 1.12.
+These updates should be backwards compatible. Sripts and notebooks should
+run and give the same results.
+
+We also actively continued to write PIG documents to finish the development and
+feature planning for Gammapy v1.0. This includes proposals for a high level
+interface, improvements in data reduction, event sampling and time analysis.
+We already started the implementation of YAML serialisation for models, event
+sampling and better lightcurve estimation. Stay tuned for the next Gammapy
+release in September!
+
+
+**Contributors:**
+
+In alphabetical order by first name:
+
+- Atreyee Sinha
+- Axel Donath
+- Brigitta Sipocz
+- Bruno Khelifi
+- Christoph Deil
+- Fabio Pintore
+- Fabio Acero
+- Kaori Nakashima
+- José Enrique Ruiz
+- Léa Jouvin
+- Luca Giunti
+- Quentin Remy
+- Régis Terrier
+- Silvia Manconi
+- Yu Wun Wong
+
+
+
+Pull Requests
++++++++++++++
+
+This list is incomplete. Small improvements and bug fixes are not listed here.
 
-The recommended conda environment for Gammapy v0.13 contains some version updates:
-Python 3.7, ipython 7.5, Scipy 1.3, Matplotlib 3.1, Astropy 3.1, healpy 1.12.
-These updates should be backward compatible, your scripts and notebooks should
-run and give the same outputs.
+See the complete `Gammapy v0.13 merged pull requests list on Github https://github.com/gammapy/gammapy/pulls?utf8=✓&q=is%3Apr+milestone%3A0.13+`__.
+
+- [#2296] Implement model YAML serialisation (Quentin Remy)
```

Comment:
I'm not a particularly big fan of this type of structuring, as it's very difficult to get any useful info out of it when one looks for reasons of a breaking piece of code in certain modules. This current layout can be easily obtained by clicking on the milestone on github and browsing the title of the PRs.

Examples for a more useful structure (imo) are e.g. astropy, scikit-learn, or even mpl.

Reply:
That is the structure we have been using so far in Gammapy, but I agree there are better solutions. For this release it probably comes too late, but we might consider a different structure for future releases.

Reply:
@bsipocz - Agreed there are better ways to write what's new and changelogs.
@adonath - Also agree to just get this out today, and then to do better with future releases.